linux/LinuxProcessList.c

```diff
@@ -1531,8 +1540,9 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
          continue;
       }
 
+      bool isMainThread = !hideUserlandThreads && !Process_isKernelThread(proc) && !parent;
```

Comment:
Maybe name the variable `scanMainThread`, since it might not be the main thread depending on the active setting?

Reply:
Seems good for me